app/src/main/java/net/programmierecke/radiodroid2/StationSaveManager.java

```diff
@@ -64,6 +64,30 @@ DataRadioStation getById(String id) {
         return null;
     }
 
+    public DataRadioStation getNextById(String id) {
+        if (listStations.size() == 0)
+            return null;
+
+        for (int i = 0; i < listStations.size() - 1; i++) {
+            if (listStations.get(i).ID.equals(id)) {
+                    return listStations.get(i + 1);
+            }
+        }
+        return listStations.get(0);
+    }
+
+    public DataRadioStation getPreviousById(String id) {
+        if (listStations.size() == 0)
```

Comment:
`if (listStations.isEmpty())`

Reply:
can i merge this?

Reply:
Not at the moment, I'll ping you when everything will be resolved.